### ISSUE TITLE
type property in a DocumentChangeAction always has "value"

### ISSUE THREAD

**Issue by meness** [angular/angularfire#2108]
### Version info

**Angular:** 8.0.0

**Firebase:** 6.1.1

**AngularFire:** 5.2.1

**Browser:** Google Chrome 75.0.3770.80

### How to reproduce these conditions

```typescript
this._firestore.collection<Model>(COLLECTION).doc<Model>(DOCUMENT).snapshotChanges()
```

### Expected behavior

`snapshotChanges()` returns a `type` property with different values: `added`, `modified`, `removed` as described [in the documentation](https://github.com/angular/angularfire2/blob/master/docs/firestore/documents.md#the-documentchangeaction-type).

### Actual behavior

`snapshotChanges()` always returns `value` for the `type` property.

**Comment by KingDarBoja** [comment]
Related to #1762

**Comment by KingDarBoja** [comment]
Seems like fixed at #2683

**Comment by jamesdaniels** [comment]
Yep. Cut in 6.1.2.
